.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: MDN Community Discussion
```

What do you think of this?

```suggestion
  - name: MDN GitHub Discussions
```